gandi/cli/commands/vhost.py

```diff
@@ -87,9 +86,10 @@ def info(gandi, resource, id):
               is_flag=True)
 @click.option('--bg', '--background', default=False, is_flag=True,
               help='Run command in background mode (default=False).')
+@click.argument('vhost', required=True)
 @pass_gandi
-def create(gandi, vhost, paas, ssl, private_key, alter_zone, poll_cert,
-           background):
+def create(gandi, paas, ssl, private_key, alter_zone, poll_cert,
+           background, vhost):
```

Comment:
you don't need to change the method signature as the parameters are passed using kwargs by click